server/finatra-server/src/main/scala/tapir/server/finatra/TapirController.scala

```diff
@@ -0,0 +1,9 @@
+package tapir.server.finatra
+import com.twitter.finatra.http.Controller
+
+trait TapirController { self: Controller =>
+  def addTapirRoute(route: FinatraRoute): Unit = {
```

Comment:
I'm not sure what's the finatra terminology, so just making sure: does it use the term "route" as well? We could also call the extension method `.toController`, if that would make sense in the finatra-world.

Reply:
Finatra does refer to individual routes as routes, but they are all built into a class which extends `Controller` which is why I named the trait `TapirController`... so I think it makes sense like this.

Reply:
sure, sounds good :)